packages/react-vapor/src/components/table-hoc/TableRowConnected.tsx

```diff
@@ -198,10 +198,8 @@ class TableRowConnected extends React.PureComponent<
         _.chain(this.props.actions)
             .filter((action: IActionOptions) => action.callOnDoubleClick)
             .forEach((action: IActionOptions) => {

```

Comment:
We could add another filter here instead of the if block down there?
```suggestion
            .fitler((action: IActionOptions) => action.enabled)
            .forEach((action: IActionOptions) => {
```

Reply:
so obvious :see_no_evil: